e2e/tests/node_not_ready/node_disconnection.robot

```diff
@@ -0,0 +1,143 @@
+*** Settings ***
+Documentation     https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-disconnection/node-disconnection/
+...
+...               Keywords are imported from the resource file
+Resource          ../../keywords/node.resource
+Resource          ../../keywords/common.resource
+Suite Setup    set_test_suite_environment
+Test setup    set_test_environment    ${TEST NAME}
```

Comment:
Shoud these also apply to existing test cases (ex: Replica Rebuilding While Replica Deletion)?